docs/api/cloudlink/packets/typing.md

```diff
@@ -0,0 +1,16 @@
+# `typing`
+
+This packet is sent when a client sends a typing indicator. These clients will be sent periodically by clients, so you need to setup a expiration system.
+### Client typing indicator frequency
+- **meo**: 3 seconds
+- **Roarer 2**: 3 seconds
+- **Meower Svelte**: 4 seconds
+
```

Comment:
```suggestion
This packet is sent when a client is currently typing.

```

again, client behavior shouldn't go here

Reply:
There should probably still be some sort of reference on what "periodically" means though.

Reply:
I disagree on this one, it's a helpful reference to have the typing times in the API docs as there is no standard for it. What if I remove the note about how to implement it but leave a general note of most clients send the packet at least every 4 seconds?

Reply:
the lack of standard is the problem here, but i'm not opposed to saying "send a typing indicator every few seconds" note

Reply:
Maybe say it expires after 4 seconds

Reply:
> Maybe say it expires after 4 seconds

Only Meower Svelte seems to use that though

Reply:
> Maybe say it expires after 4 seconds



> > Maybe say it expires after 4 seconds
> 
> Only Meower Svelte seems to use that though

I like the 4s idea, all the other ones seem to be less than 4s so 4s would work.

Reply:
does the 4s thing work for you @williamhorning?

Reply:
@Finley224 either saying "every four seconds" or saying "at an interval less than five seconds" is fine